x-pack/plugins/security_solution/cypress/integration/detection_rules/indicator_match_rule.spec.ts

```diff
@@ -487,6 +497,65 @@ describe('indicator match', () => {
       });
     });
 
+    describe('Investigate alert in timeline', () => {
+      const expectedFieldsColumns = [
+        'threat.indicator.matched.type',
+        'threat.indicator.matched.field',
+        'threat.indicator.matched.atomic',
+      ];
+
+      const expectedFieldsText = [
+        newThreatIndicatorRule.type,
+        newThreatIndicatorRule.indicatorMappingField,
+        newThreatIndicatorRule.atomic,
+      ];
+
+      before(() => {
+        cleanKibana();
+        createIndicatorMatchTimelineTemplate(newThreatIndicatorRule.timeline);
+        loginAndWaitForPageWithoutDateRange(DETECTIONS_URL);
+        goToManageAlertsDetectionRules();
+        createCustomIndicatorRule(newThreatIndicatorRule);
+        reload();
+        goToRuleDetails();
+        waitForAlertsToPopulate();
+        investigateFirstAlertInTimeline();
+      });
+
+      it('Displays the custom data providers', () => {
+        cy.get(PROVIDER_BADGE).should(
+          'have.text',
+          `threat.indicator.matched.atomic: "${newThreatIndicatorRule.atomic}"threat.indicator.matched.field: "${newThreatIndicatorRule.indicatorMappingField}"threat.indicator.matched.type: "${newThreatIndicatorRule.type}"`
+        );
+      });
+
+      it('Displays the custom columns', () => {
```

Comment:
You are totally right about the multi-assertion test. In this specific case I decided to split it in order to make easier to understand what are we asserting. But if you think that it is better to collapse it, I'm find with it :) 

In this specific case we are not resetting between `its` just because for the assertion we don't need to modify the state.